duet/views/google_uac_android_activation.view.lkml

```diff
@@ -0,0 +1,123 @@
+view: google_uac_android_activation {
+
+  derived_table: {
+    sql: WITH uac AS (
+         SELECT
+           date,
+           REGEXP_REPLACE(campaign_name, '(.*) - NULL', '\\1') as campaign,
+           SUM(spend) as cost
+         FROM
+           mozdata.google_ads.daily_campaign_stats
+         WHERE
+            campaign_name NOT LIKE '%iOS%'
+            AND date >= '2022-12-01'
+         GROUP BY
+            1, 2
+        ), activation AS (
+         SELECT
+           first_seen_date as date,
+           REGEXP_REPLACE(adjust_campaign, '([\\w]+).*', '\\1') as campaign,
+           SUM(activated) as activated,
+           COUNT(*) as new_profiles,
+         FROM `moz-fx-data-shared-prod.fenix.new_profile_activation`
+         WHERE submission_date >= '2022-12-01'
+           AND first_seen_date >= '2022-12-01'
+           AND adjust_network = 'Google Ads ACI'
+           AND adjust_campaign LIKE 'Mozilla_%'
+         GROUP BY 1, 2
+        )
+        SELECT
+          date,
+          campaign,
+          CASE WHEN campaign LIKE '%_US_%' OR campaign LIKE '%_CA_%' OR campaign LIKE '%NA%' THEN 'NA' ELSE 'EU' END AS region,
+          REGEXP_EXTRACT(campaign, '.*(Event\\d).*') as event,
+          REGEXP_EXTRACT(campaign, '.*(Group\\d).*') as group_number,
+          REGEXP_EXTRACT(campaign, 'Mozilla_FF_UAC_[\\w]{2}_([\\w]{2})_[\\w]{2}_Group\\d_Event\\d') as country_code,
+          SUM(new_profiles) as new_profiles,
+          SUM(activated) as activated,
+          SUM(cost) as cost
+        FROM uac
+        LEFT JOIN activation USING (date, campaign)
+        WHERE date < DATE_ADD(CURRENT_DATE(), INTERVAL -7 DAY)
+        GROUP BY date, campaign
+    ;;
+  }
+
+  dimension: region {
+    description: "Region of the campaign: NA or EU"
+    type: string
+    sql: ${TABLE}.region ;;
+  }
+
+  dimension: event {
+    description: "Event number used for campaign optimization"
+    type: string
+    sql: ${TABLE}.event ;;
+  }
+
+  dimension: group {
+    description: "Group of locations targeted in the campaign"
+    type: string
+    sql: ${TABLE}.group_number ;;
+  }
+
+  dimension: country_code {
+    description: "Country code from campaign name"
+    type: string
+    sql: ${TABLE}.country_code ;;
+  }
+
+  dimension: campaign {
+    description: "Campaign name"
+    type: string
+    sql: ${TABLE}.campaign ;;
+  }
+
+  dimension_group: campaign_date {
+    description: "Date of campaign spend"
+    type: time
+    datatype: date
+    timeframes: [date, week, month, year]
+    sql: ${TABLE}.date ;;
+  }
+
+  measure: new_profiles {
```

Comment:
appending `_count` might make this (and `activated` -> `activated_count`) easier to understand (especially b/c it's a `sum` measure, which might be confusing at first glance).